mk/sm.spec.in

```diff
@@ -237,13 +237,16 @@ Manager and some other packages
 %{_unitdir}/fairlock@.service
 %{_libexecdir}/fairlock
 
-%posttrans fairlock
+%post fairlock
 ## On upgrade, shut down existing lock services so new ones will
 ## be started. There should be no locks held during upgrade operations
```

Comment:
What allows to ensure that there's no lock at the time of the RPM update?

Reply:
The toolstack assures us they do not perform RPM upgrades while simultaneously performing storage management operations. Anyone on the command line who chooses to do an RPM upgrade in the middle of a disk shuffle should expect many more problems than just the odd dropped lock :-)

Reply:
Thanks. Is it the fact that the host is disabled during the upgrades which ensures this?